runtime/gc_modron_startup/mminit.cpp

```diff
@@ -1558,7 +1558,9 @@ combinationMemoryParameterVerification(J9JavaVM *javaVM, IDATA* memoryParameters
 	bool opt_XmosSet = (-1 != memoryParameters[opt_Xmos]);
 	bool opt_XmoxSet = (-1 != memoryParameters[opt_Xmox]);
 	bool opt_XsoftmxSet = (-1 != memoryParameters[opt_Xsoftmx]);
+#if defined(J9VM_GC_COMPRESSED_POINTERS)
```

Comment:
What is the documentation for this option?

Reply:
```
Sets an initial size for an area in memory that is reserved for any native classes, monitors, and threads that are used by compressed references within the lowest 4 GB memory area.
```
https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/openj9/xmcrs/index.html